modules/e2ee/E2EEncryption.js

```diff
@@ -128,8 +130,20 @@ export class E2EEncryption {
             return;
         }
 
+        if (this._enabling) {
+            await this._enabling;
+        } else {
+            this._enabling = new Deferred();
```

Comment:
You should unconditionally create the Deferred. If we awaited the previous one it means it's resolved, but if *another* call comes in, we want to also await that one.